i18n/de.toml

```diff
@@ -340,7 +340,7 @@ label = "Design"
 light = "Hell"
 
 [settings.unit]
-km = "Km"
+km = "km"
```

Comment:
Ist ne style-Frage. Beides geht, haben uns aber bewusst für das Großgeschriebene entschieden.

Reply:
Da wäre ich auf der Physikseite. Km ist keine Einheit.

Reply:
Mir geht es hier um eine konsistente Schreibweise im Settings-Dialog. Wenn wir "km" als Einheit an Zahlen verwenden schreiben wir das durchgängig wie erwartet klein. Der Text um den es hier geht ist ja eine Buttonbeschriftung und diese schreiben wir im Deutschen konsistent groß. Siehe Screen:

![Bildschirmfoto 2023-06-16 um 23 56 26](https://github.com/evcc-io/evcc/assets/152287/9d9ea515-5776-443c-9835-00d8eb1ea217)

Reply:
Auch dann gibts die Einheit nicht. Kann gerne so bleiben, aber meine naturwissenschaftliches ich zuckt bei dem Anblick.

Reply:
Das Konsistenz-Argument kann ich grundsätzlich nachvollziehen. Lösungsvorschlag in diesem Sinne: Nachdem im Settings-Dialog „Meilen“ ausgeschrieben und nicht abgekürzt (z. B. „mi“) geschrieben ist, könnte man hier – ganz im Sinne einer konsistenten Schreibweise – „Kilometer“ ebenfalls unabgekürzt ausschreiben (sofern der Platz ausreicht). Dann wäre das „Km“-„km“-Problem elegant umschifft.

Fest vorgegeben sind jedenfalls die SI-Präfixe: https://de.wikipedia.org/wiki/Vors%C3%A4tze_f%C3%BCr_Ma%C3%9Feinheiten#SI-Pr%C3%A4fixe
Die korrekte Abkürzung von „Kilometer“ lautet _immer_ „km“ (klein und aufrecht geschrieben); „Km“ hingegen ist immer falsch, da gibt es im Internationalen Einheitensystem keine Toleranz: https://de.wikipedia.org/wiki/Meter#km

Reply:
Das ausgeschriebene "Kilometer" bekommen wir an der Stelle nicht unter. Das würde dann ausgepunktet. Bin mir nicht sicher ob "Kilome..." wirklich nutzerfreundlicher wäre.

Wenn wir auf "km" (klein) gehen, müssen wir konsequenterweise auch bei Meilen zu "mi" wechseln. Damit kann ich mich anfreunden. Konsequenz wäre dann aber, dass wir die Buttonbeschriftungen aus der Übersetzung komplett rausnehmen und die Werte "km/mi" fest in die Software packen damit es für alle Sprachen gleich ist. Passt für euch?

Reply:
"Kilometer" passt doch gerade noch rein. Daher lasse ich das weiterhin als Übersetzungstexte. Das ist auch schöner für andere Zeichensätze. Für Deutsch ists jetzt in Weblate angepasst.